index.bs

```diff
@@ -690,7 +690,7 @@ times</a>. This allows a single worklet animation to coordinate multiple effects
 [[#example-2]] for an example of such a use-case.
 
 <xmp class='idl'>
-
+[Exposed=Window]
```

Comment:
s/Window/AnimationWorklet/
Because at the moment this interface is not meant to be exposed outside AnimationWorklet context.

Reply:
Great, my naive patching found a problem :)